query-engine/driver-adapters/executor/src/testd.ts

```diff
@@ -235,41 +242,40 @@ function respondOk(requestId: number, payload: unknown) {
 }
 
 type InitQueryEngineParams = {
-  engineType: ExternalTestExecutor;
-  driverAdapterManager: DriverAdaptersManager;
-  url: string;
-  schema: string;
-  logCallback: qe.QueryLogCallback;
+  env: Env
+  driverAdapterManager: DriverAdaptersManager
+  url: string
+  schema: string
+  logCallback: qe.QueryLogCallback
 };
 
 async function initQe({
-  engineType,
+  env,
   driverAdapterManager,
   url,
   schema,
   logCallback,
 }: InitQueryEngineParams) {
-  if (process.env.EXTERNAL_TEST_EXECUTOR === "Mobile") {
-    if (process.env.MOBILE_EMULATOR_URL) {
-      url = process.env.MOBILE_EMULATOR_URL;
-    }
+  if (env.EXTERNAL_TEST_EXECUTOR === 'Mobile') {
+    url = env.MOBILE_EMULATOR_URL;
```

Comment:
This unrelated condition `if (process.env.MOBILE_EMULATOR_URL) {` was removed. Maybe there was a reason for it in the first place? @SevInf might know?